node/node.go

```diff
@@ -596,36 +604,14 @@ func (n *Node) initNetworking(reg prometheus.Registerer) error {
 		}
 	}
 
-	numBootstrappers := n.bootstrappers.Count(constants.PrimaryNetworkID)
-	requiredConns := (3*numBootstrappers + 3) / 4
-
-	if requiredConns > 0 {
-		onSufficientlyConnected := make(chan struct{})
+	if requiredConns := n.getRequiredConns(); requiredConns > 0 {
```

Comment:
This is probably just mean being paranoid... But the initialization flow is pretty complex (and I worry that we might introduce a weird bug here in the future by modifying `n.bootstrappers` between here and `Start`)... 

Would it be weird to just create `onSufficientlyConnected` regardless of the value of `requiredConns`, and close it if `requiredConns <= 0`?